azure-container-apps/README.md

```diff
@@ -0,0 +1,330 @@
+
+## Step 5: Test your SCIM bridge
+
+To test if your SCIM bridge is online, choose **Overview** in your application's sidebar, then click **Application Url**. This URL is your SCIM bridge domain. You should be able to enter your bearer token to verify that your SCIM bridge is up and running.
+
+## Step 6: Connect your identity provider
+
+To finish setting up automated user provisioning, [connect your identity provider to the SCIM bridge](https://support.1password.com/scim/#step-3-connect-your-identity-provider).
+
+### If Google Workspace is your identity provider
+
+Follow the steps in this section to connect your SCIM bridge to Google Workspace.
+
+<details>
+<summary>Connect Google Workspace using the Azure Portal</summary>
+
+#### 6.1: Get your Google Workspace service account key
+
+Follow the steps to [create a Google service account, key, and API client](https://support.1password.com/scim-google-workspace/#step-1-create-a-google-service-account-key-and-api-client).
+
+#### 6.2: Download and edit the Google Workspace settings template
+
+1. Download the [`workspace-settings.json`](./google-workspace/workspace-settings.json) file from this repo.
+2. Edit the following in this file:
+    1. **Actor**: Enter the email address of the Google Workspace administrator for the service account.
+    2. **Bridge Address**: Enter your SCIM bridge domain. This is the Application URL for your Container App, found on the overview page (**not** your 1Password account sign-in address). For example: `https://op-scim-bridge.example.eastus.azurecontainerapps.io`.
+3. Save the file.
+
+#### 6.3: Create secrets for Google Workspace
+
+1. Open the Azure Portal and go to the [Container Apps](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps) page.
+2. Choose **Secrets** from the Settings section in the sidebar.
+3. Open the key file generated by Google for your service account you saved to your computer in [6.1](#61-get-your-google-workspace-service-account-key).
+4. Click **Add** to create a secret for the credentials file, then fill out the following fields:
+    1. **Key**: Enter `workspace-credentials`.
+    2. **Value**: Copy the entire contents of the key file. Paste it into this field.
+5. Click **Add**, then wait until the secret is created.
+6. Open the settings file you edited in [6.2](#62-download-and-edit-the-google-workspace-settings-template).
+7. Click **Add** to create another secret for the settings file, then fill out the following fields:
+    1. **Key**: Enter `workspace-settings`.
+    2. **Value**: Copy the entire contents of the settings file. Paste it into this field.
+8. Click **Add**, then wait until the secret is created.
+
+#### 6.4: Connect your SCIM Bridge to Google Workspace
+
+1. Choose **Containers** from the Application section in the sidebar.
+2. Click **Edit and deploy**
+3. Choose **Volumes**.
+4. Hover over the **credentials** volume and select it to edit the volume.
```

Comment:
```suggestion
4. Click on the **credentials** volume listed to edit the volume.
```

Reply:
I think this is how it should be to align with other steps in the doc and our style guide:

```suggestion
4. Hover over the **credentials** volume and click it to edit the volume.
```

Reply:
@Penderworth Leaving this one to you to align with our best practices style guide wise. 
![credentials step](https://github.com/1Password/scim-examples/assets/106387451/844b1ea8-100b-4c19-822a-2562858423e7)